src/pineko/evolve.py

```diff
@@ -76,54 +82,80 @@ def evolve_grid(
     fktable_path,
     max_as,
     max_al,
+    xir,
+    xif,
+    alphas_values=None,
     assumptions="Nf6Ind",
     comparison_pdf=None,
 ):
-    """
-    Convolute grid with EKO from file paths.
+    """Convolute grid with EKO from file paths.
 
     Parameters
     ----------
-        pineappl_path : str
-            unconvoluted grid
-        eko_path : str
-            evolution operator
-        fktable_path : str
-            target path for convoluted grid
-        max_as : int
-            maximum power of strong coupling
-        max_al : int
-            maximum power of electro-weak coupling
-        comparison_pdf : None or str
-            if given, a comparison table (with / without evolution) will be printed
+    pineappl_path : str
+        unconvoluted grid
+    eko_path : str
+        evolution operator
+    fktable_path : str
+        target path for convoluted grid
+    max_as : int
+        maximum power of strong coupling
+    max_al : int
+        maximum power of electro-weak coupling
+    xir : float
+        renormalization scale variation
+    xif : float
+        factorization scale variation
+    alphas_values : None or list
+        values of strong coupling used to collapse grids
+    assumptions : str
+        assumtions on the flavor dimension
```

Comment:
assumptions :)

Reply:
Just because @felixhekhorn likes to be able to click on the button...